internal/pkg/deploy/cloudformation/stack/workload.go

```diff
@@ -264,6 +264,134 @@ func validatePath(input string, maxLength int) error {
 	return nil
 }
 
+func validateStorageConfig(in *manifest.Storage) error {
+	if in == nil {
+		return nil
+	}
+	return validateVolumes(in.Volumes)
+}
+
+func validateVolumes(in map[string]manifest.Volume) error {
+	for name, v := range in {
+		if err := validateVolume(name, v); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func validateVolume(name string, in manifest.Volume) error {
+	if err := validateMountPointConfig(in); err != nil {
+		return fmt.Errorf("validate container configuration for volume %s: %w", name, err)
```

Comment:
nit: can we remove wrappers if we are calling private functions since we already return clear error messages from them